flink-streamlit/app.py

```diff
@@ -47,7 +45,7 @@ async def display_quotes(component):
     topic_name = option
 
     # starting from a specific partition here, it may be different depending on the topic so try a few out or just start from the beginning with the auto.offset.reset config
-    partition = TopicPartition(f"tumble_interval_{topic_name}", 0, 7)
+    partition = TopicPartition(f"tumble_interval_{topic_name}", 0, "OFFSET_BEGINNING")
     consumer.assign([partition])
     consumer.seek(partition)
```

Comment:
It might be cleaner to replace 49-52 with 
```
topic_name = f"tumble_interval_{option}"
consumer.subscribe(topic_name)
```
Then no one has to worry about which partition to read from.